src/odemis/gui/xmlh/resources/panel_tab_cryosecom_chamber.xrc

```diff
@@ -8,8 +8,7 @@
 					<object class="wxBoxSizer">
 						<object class="sizeritem">
 							<object class="wxPanel" name="pnl_project">
-								<object class="wxBoxSizer">
-									<orient>wxVERTICAL</orient>
+								<object class="wxGridBagSizer">
```

Comment:
Can you try fixing the indentation for lines? I tried fixing it during the refactoring and failed haha

Reply:
I think it's indented correctly, but with tabs instead of 2 spaces so it displays weird in github?